lib/cmd/fhc/target.js

```diff
@@ -104,32 +97,6 @@ function target(argv, cb) {
         ini.set(keys.KEY_ID, targ[keys.KEY_ID], "user");
       }
 
-      var _gotVersion = function() {
-        ini.save(function(err) {
-          var message;
-          if (targ.user) {
-            message = " User: " + targ.user;
-          } else {
-            message = " API Key: " + targ[keys.KEY_ID];
-          }
-          target.message = i18n._("Successfully targeted: ") + targ.target + message;
-          return cb(err, targ);
-        });
-      };
-
-      ngui({_ : []}, function(err, isFh3) {
-        if (err) {
-          if (targ.fhversion) {
-            ini.set('fhversion', targ.fhversion);
-          } else {
-            target.message += i18n._('Your login has expired - please log in again!\n ') + login.usage;
-          }
-          return _gotVersion();
-        }
-        var version = (isFh3) ? 3 : 2;
-        ini.set('fhversion', version);
-        return _gotVersion();
-      });
```

Comment:
Hi @camilamacedo86 , I keep getting that error. My steps to reproduce are as follows:
mkdir ~/tmp
cd ~/tmp
git clone git@github.com:camilamacedo86/fh-fhc.git
cd fh-fhc
git checkout FH-3618
npm install
./bin/fhc.js target https://testing.zeta.feedhenry.com/

Result: fhc ERR! cb() never called!